qemu/tests/blockdev_inc_backup_no_bitmap.py

```diff
@@ -0,0 +1,78 @@
+from virttest.qemu_monitor import QMPCmdError
+
+from provider import backup_utils
+from provider import blockdev_base
+
+
+class BlkdevIncNobitmap(blockdev_base.BlockdevBaseTest):
+
+    def __init__(self, test, params, env):
+        super(BlkdevIncNobitmap, self).__init__(
+            test, params, env)
+        self.source_images = []
+        self.full_backups = []
+        self.inc_backups = []
+        self.bitmaps = []
+        self.src_img_tags = params.objects("source_images")
+        self.inc_sync_mode = params["inc_sync_mode"]
+        list(map(self._init_arguments_by_params, self.src_img_tags))
+
+    def _init_arguments_by_params(self, tag):
+        image_params = self.params.object_params(tag)
+        image_chain = image_params.objects("image_backup_chain")
+        self.source_images.append("drive_%s" % tag)
+        self.full_backups.append("drive_%s" % image_chain[0])
+        self.inc_backups.append("drive_%s" % image_chain[1])
+        self.bitmaps.append("bitmap_%s" % tag)
+
+    def do_full_backup(self):
+        extra_options = {"sync": "full", "auto_disable_bitmap": False}
```

Comment:
auto_disable_bitmap is not used, we can remove it.

Reply:
have update, thanks zhenchao.